### DISCUSSION TITLE
建议：鼠标休眠后唤醒后晃动鼠标放大鼠标箭头

### DISCUSSION THREAD

**Discussion by ghostzhao** [shuax/MouseInc.Settings#270] [Ideas]
类似与mac 晃动鼠标  箭头放大 或者Shake to Find Cursor 软件的功能，鼠标休眠后 唤醒  晃动鼠标放大鼠标指针 或者变色寻找鼠标位置
，比如开的网页白色，鼠标箭头也是白色 屏幕过大 休眠后晃好几下 才知道鼠标坐标位置。

**Comment by shuax** [comment]
https://github.com/microsoft/PowerToys/

建议用这个
